spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -260,16 +260,19 @@ def create!(name: nil, bundle_id: nil, certificate: nil, devices: [], mac: false
         #  If you're calling this from a subclass (like AdHoc), this will
         #  only return the profiles that are of this type
         # @param mac (Bool) (optional): Pass true to get all Mac provisioning profiles
-        # @param xcode DEPRECATED
+        # @param xcode (Bool) (optional): Pass true to include Xcode managed provisioning profiles
```

Comment:
Should we mention this won't work if you have set `SPACESHIP_AVOID_XCODE_API` and add the warning output back in for that case?